packages/styles/src/themes/bootstrap/_overrides-color.scss

```diff
@@ -20,11 +20,23 @@ $secondary: color.$gray-60;
 $info: color.$yellow;
 // $warning: color.$warning; // Is already defined as $warning in post-colors
 $danger: color.$error;
-$light: color.$gray-background-light;
-$dark: $primary;
 
 // Colors that can be used as a background
-$theme-colors: color.$background-colors !default;
+$colors: () !default;
+$colors: map.merge(
+  $colors,
+  (
+    'primary': $primary,
+    'secondary': $secondary,
+    'white': color.$white,
+    'light': color.$light,
+    'dark': color.$dark,
+  )
+);
+$colors: map.merge($colors, color.$contextual-colors);
+
+$theme-colors: () !default;
+$theme-colors: map.merge($theme-colors, $colors);
```

Comment:
This lets components with theme-backgrounds work again, but it causes another problem...
Now we have a lot of colors in the :root selector, declaired by bootstrap. And they do not all reflect the post-colors (e.g. --bs-gray-*).
This can be confusing for the devs using the Swiss Post Design System.
Maybe we should think about replacing bootstraps theme-colors variables completly from the project.

At least it would be nice, to get rid of all the color-variables from bootstrap.
![image](https://user-images.githubusercontent.com/9716662/185870574-bca3c3c5-e04c-437c-8e86-4d7d2c53108f.png)

What do you think about that?